### PR TITLE
Render Chessboard for as viewed by player with black pieces.

### LINKED ISSUE #65: Render Chess board as viewed player with black pieces
Currently, the chess board is only rendered as viewed by the player with white pieces.

### PR DESCRIPTION
**Pull request by codemayank**
fixes #65 
chessBoardB

### CODE REVIEW

**Comment by Prabhakar-Poudel** [comment]
@codemayank 
Can't we achieve this by rotating board 180 deg using css (which will be 2-3 lines of code) for black player, instead of all JS solution?

**Comment by codemayank** [comment]
Yes, we can definitely achieve this by using CSS transforms. However, it will be more than 2-3 lines of code as we also will have to rotate all the chess pieces and the timers. I feel this will unnecessarily make it more complicated and will also be hard to write tests.

**Comment by codemayank** [comment]
The CSS solution could be more complicated as we have to use CSS transform on all components (timer, chess pieces, board, etc.) and it will also be difficult to write tests for it.

File: client/src/components/chess-game/ChessGame.js [hunk 1]
```diff
@@ -14,6 +14,15 @@ import './ChessGame.css';
  * and while running server or building server it exports function
 */
 const Chess = typeof ChessJs === 'function' ? ChessJs : ChessJs.Chess;
+const blackPlayerBoard = (squares, board) => {
+  const newSquares = Object.assign([], squares);
+  let newBoard = Object.assign([], board.map(x => Object.assign([], x)));
```
Comment (Prabhakar-Poudel) [inline client/src/components/chess-game/ChessGame.js]: What is x?

Comment (Prabhakar-Poudel) [inline client/src/components/chess-game/ChessGame.js]: Are you creating a copy of board array here? if so, board.map will return a new array already.

File: client/src/components/chess-game/ChessGame.js [hunk 2]
```diff
@@ -14,6 +14,15 @@ import './ChessGame.css';
  * and while running server or building server it exports function
 */
 const Chess = typeof ChessJs === 'function' ? ChessJs : ChessJs.Chess;
+const blackPlayerBoard = (squares, board) => {
+  const newSquares = Object.assign([], squares);
```
Comment (Prabhakar-Poudel) [inline client/src/components/chess-game/ChessGame.js]: You can use Array.from(squares) to create new array, which is more intuitive.

**Review by Prabhakar-Poudel — APPROVED** [review]
LGTM after fixing comments.

File: client/src/components/chess-game/ChessGame.js (hunk 1 above)
Comment (Prabhakar-Poudel) [reply client/src/components/chess-game/ChessGame.js]: Write name if it signifies an entity.

File: client/src/components/chess-game/ChessGame.js (hunk 2 above)
Comment (codemayank) [reply client/src/components/chess-game/ChessGame.js]: Done.

File: client/src/components/chess-game/ChessGame.js (hunk 1 above)
Comment (codemayank) [reply client/src/components/chess-game/ChessGame.js]: Done.

Comment (codemayank) [reply client/src/components/chess-game/ChessGame.js]: x is for rows of the chessboard grid. I have renamed it to row.

**Pull request merged**